src/testRunner/unittests/convertToAsyncFunction.ts

```diff
@@ -1178,11 +1188,17 @@ function [#|f|]() {
     }
 `);
 
+    _testConvertToAsyncFunction("convertToAsyncFunction_simpleFunctionExpression", `
+const [#|foo|] = function () {

```

Comment:
>[#|foo|] [](start = 6, length = 8)

I realize you didn't change this, but that seems like a weird place for the squiggle.  What if the function had a name?

Reply:
Tracked at #26801.